Engine/Results/BaseResultsHandler.cs

```diff
@@ -106,6 +106,16 @@ public abstract class BaseResultsHandler
         /// </summary>
         protected AlphaRuntimeStatistics AlphaRuntimeStatistics { get; set; }
 
+        /// <summary>
+        /// Closing portfolio value. Used to calculate daily performance.
+        /// </summary>
+        protected decimal _dailyPortfolioValue;
+
+        /// <summary>
+        /// Last time the <see cref="IResultHandler.Sample(DateTime, bool)"/> method was called in UTC
+        /// </summary>
+        protected DateTime _previousUtcSampleTime;
```

Comment:
nit - since they are now `protected` they should be renamed to `DailyPortfolioValue` and `PreviousUtcSampleTime`